inventory/sample/group_vars/k8s_cluster/k8s-net-calico.yml

```diff
@@ -59,7 +59,7 @@
 # calico_datastore: "kdd"
 
 # Choose Calico iptables backend: "Legacy", "Auto" or "NFT"
-# calico_iptables_backend: "Legacy"
+# calico_iptables_backend: "Auto"
```

Comment:
hum and why are you keeping the default to Legacy in https://github.com/kubernetes-sigs/kubespray/blob/caff539ccd91d22bd26c6cdba59da0225ee3bb0a/roles/network_plugin/calico/defaults/main.yml#L81

Reply:
Because I forgot to grep for it :). Now it's set.

Reply:
nice catch @floryut :-)